types/react-native-elements/index.d.ts

```diff
@@ -1,11 +1,60 @@
-// Type definitions for react-native-elements 0.13
+// Type definitions for react-native-elements 0.16
 // Project: https://github.com/react-native-training/react-native-elements#readme
 // Definitions by: Kyle Roach <https://github.com/iRoachie>
+//                 Ifiok Jr. <https://github.com/ifiokjr>
```

Comment:
Adding your name here means you become a maintainer of the project which means you'll automatically get email/notified of all issues and PRs for this lib. Just making sure you know what it means

Reply:
That's okay with me.